src/components/jobCard.js

```diff
@@ -21,7 +22,7 @@ export default function JobCard({
       <div className="md:flex-shrink-0">
         <img
           className="h-48 w-full object-cover md:w-48"
-          src={boat}
+          src={(img)? img: boat}
```

Comment:
I think we should replace the boat with a different image if the thumbnail is missing, possibly this one?
![no_image](https://github.com/J-Raymer/printlink3d/assets/57274383/8b3791cb-a15c-48c8-bbaf-8835a503d0fe)